flink/v1.15/flink/src/main/java/com/netease/arctic/flink/read/FlinkSplitPlanner.java

```diff
@@ -132,17 +128,15 @@ public static List<ArcticSplit> mergeOnReadPlan(
     return morSplits;
   }
 
-  public static List<ArcticSplit> planChangeTable(TableEntriesScan tableEntriesScan, Long fromSequence,
-                                                  PartitionSpec spec, AtomicInteger splitCount) {
-    CloseableIterable<IcebergFileEntry> entries = tableEntriesScan.entries();
-
-    BaseAndChangeTask baseAndChangeTask = BaseAndChangeTask.ofIceberg(entries, spec, fromSequence);
-
+  public static List<ArcticSplit> planChangeTable(ChangeTableIncrementalScan tableIncrementalScan, Long fromSequence,
```

Comment:
I think we could remove the `fromSequence` variable. 
Add this `changeTableScan.fromSequence(fromSequence);` before invoke the planChangeTable() method.

Reply:
SGTM.
I have removed it, TAL.